evm/src/main/java/org/hyperledger/besu/evm/code/OpcodesV1.java

```diff
@@ -0,0 +1,327 @@
+    VALID, // 0x8c - DUP13
+    VALID, // 0x8d - DUP14
+    VALID, // 0x8e - DUP15
+    VALID, // 0x8f - DUP16
+    VALID, // 0x90 - SWAP1
+    VALID, // 0x91 - SWAP2
+    VALID, // 0x92 - SWAP3
+    VALID, // 0x93 - SWAP4
+    VALID, // 0x94 - SWAP5
+    VALID, // 0x95 - SWAP6
+    VALID, // 0x96 - SWAP7
+    VALID, // 0x97 - SWAP8
+    VALID, // 0x98 - SWAP9
+    VALID, // 0x99 - SWAP10
+    VALID, // 0x9a - SWAP11
+    VALID, // 0x9b - SWAP12
+    VALID, // 0x9c - SWAP13
+    VALID, // 0x9d - SWAP14
+    VALID, // 0x9e - SWAP15
+    VALID, // 0x9f - SWAP16
+    VALID, // 0xa0 - LOG0
+    VALID, // 0xa1 - LOG1
+    VALID, // 0xa2 - LOG2
+    VALID, // 0xa3 - LOG3
+    VALID, // 0xa4 - LOG4
+    INVALID, // 0xa5
+    INVALID, // 0xa6
+    INVALID, // 0xa7
+    INVALID, // 0xa8
+    INVALID, // 0xa9
+    INVALID, // 0xaa
+    INVALID, // 0xab
+    INVALID, // 0xac
+    INVALID, // 0xad
+    INVALID, // 0xae
+    INVALID, // 0xaf
+    INVALID, // 0xb0
+    INVALID, // 0xb1
+    INVALID, // 0xb2
+    INVALID, // 0xb3
+    INVALID, // 0xb4
+    INVALID, // 0xb5
+    INVALID, // 0xb6
+    INVALID, // 0xb7
+    INVALID, // 0xb8
+    INVALID, // 0xb9
+    INVALID, // 0xba
+    INVALID, // 0xbb
+    INVALID, // 0xbc
+    INVALID, // 0xbd
+    INVALID, // 0xbe
+    INVALID, // 0xbf
+    INVALID, // 0xc0
+    INVALID, // 0xc1
+    INVALID, // 0xc2
+    INVALID, // 0xc3
+    INVALID, // 0xc4
+    INVALID, // 0xc5
+    INVALID, // 0xc6
+    INVALID, // 0xc7
+    INVALID, // 0xc8
+    INVALID, // 0xc9
+    INVALID, // 0xca
+    INVALID, // 0xcb
+    INVALID, // 0xcc
+    INVALID, // 0xcd
+    INVALID, // 0xce
+    INVALID, // 0xcf
+    INVALID, // 0xd0
+    INVALID, // 0xd1
+    INVALID, // 0xd2
+    INVALID, // 0xd3
+    INVALID, // 0xd4
+    INVALID, // 0xd5
+    INVALID, // 0xd6
+    INVALID, // 0xd7
+    INVALID, // 0xd8
+    INVALID, // 0xd9
+    INVALID, // 0xda
+    INVALID, // 0xdb
+    INVALID, // 0xdc
+    INVALID, // 0xdd
+    INVALID, // 0xde
+    INVALID, // 0xef
+    INVALID, // 0xe0
+    INVALID, // 0xe1
+    INVALID, // 0xe2
+    INVALID, // 0xe3
+    INVALID, // 0xe4
+    INVALID, // 0xe5
+    INVALID, // 0xe6
+    INVALID, // 0xe7
+    INVALID, // 0xe8
+    INVALID, // 0xe9
+    INVALID, // 0xea
+    INVALID, // 0xeb
+    INVALID, // 0xec
+    INVALID, // 0xed
+    INVALID, // 0xee
+    INVALID, // 0xef
+    VALID, // 0xf0 - CREATE
+    VALID, // 0xf1 - CALL
+    VALID, // 0xf2 - CALLCODE
+    VALID_AND_TERMINAL, // 0xf3 - RETURN
+    VALID, // 0xf4 - DELEGATECALL
+    VALID, // 0xf5 - CREATE2
+    INVALID, // 0xf6
+    INVALID, // 0xf7
+    INVALID, // 0xf8
+    INVALID, // 0xf9
+    VALID, // 0xfa - STATICCALL
+    INVALID, // 0xfb
+    INVALID, // 0xfc
+    VALID_AND_TERMINAL, // 0xfd - REVERT
+    VALID_AND_TERMINAL, // 0xfe - INVALID
+    VALID_AND_TERMINAL, // 0xff - SELFDESTRUCT
+  };
+
+  private OpcodesV1() {
+    // static utility class
+  }
+
+  static long[] validateAndCalculateJumpDests(final Bytes code) {
```

Comment:
Can we add a reference to the [EIP-3670](https://eips.ethereum.org/EIPS/eip-3670) here?

Reply:
I'll rename it and add all validation EIPS as doc references